app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +30,196 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdge()
         setContent {
             Kuit6_week1Theme {
-                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
-                    Greeting(
-                        name = "Android",
-                        modifier = Modifier.padding(innerPadding)
-                    )
-                }
+
             }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
-        modifier = modifier
+fun NameCard(modifier: Modifier = Modifier) {
+    Column(
+        modifier = Modifier.padding(top = 45.dp)
+    ) {
+        SampleRow(modifier = Modifier.padding(top = 20.dp)) {
+            Box(
+                modifier
+                    .size(240.dp, 140.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(16.dp)
+                    )
+                    .padding(10.dp),
+                //verticalAlignment = Alignment.CenterVertically 여기서 이걸 쓰고싶은데 왜 이걸 쓰자마자 다른 함수들에서 오류가 발생하는지 모르겠습니다..
+            ) { //박스 함수의 시작점.
+                Image(
+                    painter = painterResource(R.drawable.image2),
+                    contentDescription = "쿠잇 로고 이미지",
+                    modifier = Modifier
+                        .size(40.dp, 40.dp)
```

Comment:
이런식으로 가로,세로의 크기값이 같은 경우엔 아래 작성해드린 것처럼 하나의값만 작성하시면 가로, 세로에 같은 값이 적용됩니다 !
```suggestion
                        .size(40.dp)
```